environments/production/terraform.yml

```diff
@@ -420,6 +420,7 @@ rds_instances:
     engine_version: "14"
     backup_window: "23:00-01:00"
     backup_retention: 7
+    iops: 18000
```

Comment:
This is per second?
18k, oh wow, that seems high.

Reply:
Yeah. This is per second.